diagnostic_msgs/KeyValue[] data
